Dockerfile

```diff
@@ -23,6 +23,8 @@ RUN git clone https://github.com/openlink/virtuoso-opensource.git \
         && cd .. \
         && rm -r /virtuoso-opensource
 
+HEALTHCHECK CMD wget -q -O - http://localhost:8890/sparql'?query=ASK%20%7B%7D' || exit 1
```

Comment:
As discussed, because we need the service to finally run on PID 1. Yes it is possible but it is unclean.